README.md

```diff
@@ -1,58 +1,64 @@
-[![Build Status](https://dev.azure.com/digitalpalireader/digitalpalireader/_apis/build/status/digitalpalireader%20-%20build?branchName=master)](https://dev.azure.com/digitalpalireader/digitalpalireader/_build/latest?definitionId=1&branchName=master) [![Release Status](https://vsrm.dev.azure.com/digitalpalireader/_apis/public/Release/badge/5c79c309-a1e5-4a73-b2a5-4faa0c2a2cce/1/2)](https://dev.azure.com/digitalpalireader/digitalpalireader/_release?_a=releases&view=mine&definitionId=1) [![License: CC BY-NC-SA 4.0](https://img.shields.io/badge/License-CC%20BY--NC--SA%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by-nc-sa/4.0/)

```

Comment:
i fixed the .vscode settings - please can you revert this file and make your change again so vscode automatically does not line endings and other stuff. that ways we can see only your changed lines.

Reply:
Ah yes, sorry about that!